cmd/agent/subcommands/jmx/command.go

```diff
@@ -112,6 +113,13 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 			}),
 			workloadmeta.Module(),
 			apiimpl.Module(),
+			fx.Provide(func(config config.Component) tagger.Params {
+				if pkgconfig.IsCLCRunner() {
+					return tagger.NewCLCRunnerRemoteTaggerParams()
+				}
+				return tagger.NewTaggerParams()
+			}),
+			tagger.Module(),
```

Comment:
I'm not sure that the JMX command need the tagger.
Can be validated by the @DataDog/agent-metrics-logs team

Reply:
I think they retrieve some metrics that might need to be tagged